code/drasil-lang/lib/Language/Drasil/CodeExpr/Lang.hs

```diff
@@ -70,6 +70,18 @@ data UFuncVV = NegV
 data UFuncVN = Norm | Dim
   deriving Eq
 
+-- | Set + Set -> Set
+data SSet = SUnion
```

Comment:
Following the pattern below, I think this would be `SSSBinOp`? Union is also associative, so you can build it similar to the other associative operations.

Reply:
Yes, I agree `___BinOp` would probably be better. For Union I was thinking of doing
data AssocConcatOper = SUnion
        deriving Eq